src/java/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDC.java

```diff
@@ -274,9 +274,12 @@ private static class CDCSizeTracker extends DirectorySizeCalculator
         // track the total size between two dictionary size calculations
         private final AtomicLong sizeInProgress;
 
+        private final File path;
+
         CDCSizeTracker(CommitLogSegmentManagerCDC segmentManager, File path)
         {
-            super(path);
+            super();
```

Comment:
why do we need this `super()` here to be called explicitly?

Reply:
Good catch. Removed it.